tests/files/c_source/motohawk.h

```diff
@@ -40,6 +40,9 @@
 
 #define MOTOHAWK_FRAME_ID_EXAMPLE_MESSAGE (0x1f0U)
 
+#define MOTOHAWK_EXAMPLE_MESSAGE_ENABLE_VAL_DISABLED (0U)
+#define MOTOHAWK_EXAMPLE_MESSAGE_ENABLE_VAL_ENABLED (1U)
```

Comment:
I think we should rename `VAL` to `CHOICE`, and possibly move `CHOICE` to the same position as `FRAME_ID` in the frame id defines. That is, `MOTOHAWK_CHOICE_EXAMPLE_MESSAGE_ENABLE_ENABLED`. Alternatively we move both `FRAME_ID` and `CHOICE` to the end of the defines to make naming consistent throughout the generated code. See the example below, where all symbols starts with `motohawk_example_message_[enable_]` or `MOTOHAWK_EXAMPLE_MESSAGE_[ENABLE_]`.

``` c
#define MOTOHAWK_EXAMPLE_MESSAGE_FRAME_ID (0x1f0U)

#define MOTOHAWK_EXAMPLE_MESSAGE_ENABLE_ENABLED_CHOICE (1U)

struct motohawk_example_message_t {
    ...
};

ssize_t motohawk_example_message_encode(...);

int motohawk_example_message_decode(...);
```